lib/wda/webdriveragent.js

```diff
@@ -119,7 +122,9 @@ class WebDriverAgent {
       if (this.prebuildWDA) {
         await this.xcodebuild.prebuild();
       }
-      return await this.xcodebuild.start();
+      return await retryInterval(WDA_LAUNCH_RETRY, WDA_LAUNCH_DELAY, async () => {
```

Comment:
How this helps to improve stability? If the source cannot be compiled because of signing problem then repeating the building won't help to cure it.

Reply:
>How this helps to improve stability? If the source cannot be compiled because of signing problem then repeating the building process won't help to cure it.

I think it just happens when we develop a test script or run appium at the first time. It isn't a big deal.

This PR is fixed `xcodebuild exited with code '65'` issue when we run a test suite (multiple test cases) in a long time:
* https://github.com/appium/appium/issues/8661
* https://github.com/facebook/WebDriverAgent/issues/507
* https://github.com/appium/appium/issues/7742#issuecomment-309367888